modules/examples/personas/personas.py

```diff
@@ -0,0 +1,610 @@
+from typing import Any
+
+from aws_cdk import Aws, Environment
+from aws_cdk import aws_iam as iam
+from constructs import Construct
+
+
+class Personas(Construct):
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        s3_bucket_prefix: str,
+        env: Environment,
+        **kwargs: Any,
+    ) -> None:
+        super().__init__(scope, construct_id)
+
+        # Define common permissions
+        sagemaker_describe_permissions = [
+            "sagemaker:DescribeModel",
+            "sagemaker:DescribeEndpoint",
+            "sagemaker:DescribeEndpointConfig",
+            "sagemaker:DescribeTrainingJob",
+            "sagemaker:DescribeHyperParameterTuningJob",
+            "sagemaker:DescribeProcessingJob",
+            "sagemaker:DescribeTransformJob",
+        ]
+
+        sagemaker_list_permissions = [
+            "sagemaker:ListModels",
+            "sagemaker:ListEndpoints",
+            "sagemaker:ListEndpointConfigs",
+            "sagemaker:ListTrainingJobs",
+            "sagemaker:ListHyperParameterTuningJobs",
+            "sagemaker:ListProcessingJobs",
+            "sagemaker:ListTransformJobs",
+        ]
+
+        s3_permissions = [
+            "s3:GetObject",
+            "s3:ListBucket",
+        ]
+
+        ecr_permissions = [
+            "ecr:GetAuthorizationToken",
+            "ecr:BatchCheckLayerAvailability",
+            "ecr:GetDownloadUrlForLayer",
+            "ecr:BatchGetImage",
+        ]
+
+        glue_describe_permissions = [
+            "glue:GetTable",
+            "glue:GetDatabase",
+            "glue:GetJob",
+            "glue:GetTrigger",
+            "glue:GetCrawler",
+            "glue:GetDevEndpoint",
+        ]
+        # ML Engineer role
+        ml_engineer_role = iam.Role(
+            self,
+            "MLEngineerRole",
+            assumed_by=iam.ServicePrincipal("sagemaker.amazonaws.com"),
+            role_name="aiops-MLEngineer",
```

Comment:
Should we keep the `project_name-deployment_name-self.module_name` prefix for these role names? I don't think hardcoding `aiops` as a prefix is a good idea. For example, this should resolve to `aiops-personas-personas-MLEngineer`.